### ISSUE TITLE
Configurable optional glyphicon-new-window link

### ISSUE THREAD

**Issue by mrgmatthews** [stoicflame/enunciate#298]
For some of our REST apis we use a identifiable object in the pathway  link/{ProductId}/product (with GETs and POSTs) the link that's generated doesn't seem to have an ability to turn if off.

`<h3><span class="resource-method">${method.label}</span> <span class="resource-path">${resource.path} <a href="${apiRelativePath}${resourceGroup.relativeContextPath}${resource.relativePath}"`

it would be nice if the <a href was only included if the ${resource.relativePath} didnt contain { and } within its value

**Comment by mrgmatthews** [comment]
something like:
<h3><span class="resource-method">${method.label}</span> <span class="resource-path">${resource.path} [#if !resource.relativePath?contains("{") && !resource.relativePath?contains("}")  ]
<a href="${apiRelativePath}${resourceGroup.relativeContextPath}${resource.relativePath}" class="glyphicon glyphicon-new-window" target="_blank"></a>
[/#if]</span></h3>

**Comment by mrgmatthews** [comment]
two pull requests, one based on pathvariables being { } and a global disable links

**Comment by stoicflame** [comment]
I'll take a look for 2.3.

**Comment by stoicflame** [comment]
Configuration option available at d76df29.

```xml
<enunciate>
  <modules>
    <docs disableResourceLinks="true"/>
```

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
